### ISSUE TITLE
Is it correct to use the pretrained R50 with unfrozen BN?

### ISSUE THREAD

**Issue by alekseynp** [facebookresearch/detectron2#1159]
Is it correct to use `detectron2://ImageNetPretrained/MSRA/R-50.pkl` with `MODEL.RESNETS.NORM BN`?

I see conflicting messaging below.

From `MODEL_ZOO.md`:
> We provide backbone models pretrained on ImageNet-1k dataset. These models have different format from those provided in Detectron: we do not fuse BatchNorm into an affine layer.

From `detectron2/checkpoint/catalog.py:13`
```python
    # NOTE: all BN models here have fused BN into an affine layer.
    # As a result, you should only load them to a model with "FrozenBN".
    # Loading them to a model with regular BN or SyncBN is wrong.
```
For what it's worth, the logging at least appears to load BN stats from the checkpoint.

Therefore I think the comments in `catalog.py` are incorrect?

**Comment by ppwwyyxx** [comment]
You can use BN not frozen, with detectron2://ImageNetPretrained/MSRA/R-50.pkl.

Code in `detectron2/checkpoint/catalog.py` is for models that start with "catalog://". They do not apply to models that start with "detectron2://".

**Comment by cheneeheng** [comment]
I was having the same question too until I found this. Perhaps include as sidenote in MODEL_ZOO.md ?
